pkgs/by-name/de/deskreen/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  appimageTools,
+  alsa-lib,
+  at-spi2-atk,
+  cairo,
+  dbus,
+  expat,
+  glib,
+  glibc,
+  gtk3,
+  hicolor-icon-theme,
+  cups,
+  libdrm,
+  xorg,
+  libxkbcommon,
+  mesa,
+  nspr,
+  nss,
+  pango
+}:
+
+stdenv.mkDerivation rec{
+  pname = "deskreen";
+  version = "2.0.4";
+
+  deskreen = appimageTools.wrapType2{
```

```suggestion
  deskreen = appimageTools.wrapType2 {
```

Please be consistent with code style. Try to make sure to add a space before the function call and its parameter.